src/pages/ReimbursementAccount/ReimbursementAccountForm.js

```diff
@@ -58,9 +58,9 @@ class ReimbursementAccountForm extends React.Component {
                     onFixTheErrorsLinkPressed={() => {
                         this.form.scrollTo({y: 0, animated: true});
                     }}
-                    message={this.props.reimbursementAccount.error}
+                    message={this.props.reimbursementAccount.error || _.values(this.props.reimbursementAccount.errors)[0]}
```

Comment:
```suggestion
                    message={this.props.reimbursementAccount.error || _.last(_.values(this.props.reimbursementAccount.errors))}
```

We want the latest error in the errors field, right?

Reply:
Ooo yeah good catch, ideally we should also sort these errors too, I actually wrote [a function](https://github.com/Expensify/App/blob/2343eeb08fbb89ec34e3de331ae1f75e494478fb/src/libs/ErrorUtils.js#L39-L52) in `ErrorUtils` a while back that does this.

I think you just use it here like so, don't forget to import it if you commit from here.

```suggestion
                    message={this.props.reimbursementAccount.error || ErrorUtils.getLastestErrorMessage(this.props.reimbursementAccount)}
```